src/interceptors/request.ts

```diff
@@ -58,6 +55,44 @@ export function defaultRequestInterceptor(axios: AxiosCacheInstance): RequestInt
       }
     }
 
+    // allowUrls
+    if (
+      typeof config.cache.cachePredicate === 'object' &&
+      config.cache.cachePredicate.allowUrls &&
+      config.url
+    ) {
+      let matched = false;
+
+      function logDebug(matched: boolean, cachePredicate: CachePredicateObject) {
```

Comment:
I think I was wrong, better not to create this inner function because when compiling in production mode, instead of removing everything, the `__ACI_DEV__` inside the function will just make it an empty function. Could you please move it back to 2 different logs?